google/cloud/spanner/integration_tests/client_integration_test.cc

```diff
@@ -350,40 +350,49 @@ TEST_F(ClientIntegrationTest, RunTransaction) {
 
 /// @test Test various forms of ExecuteSql()
 TEST_F(ClientIntegrationTest, ExecuteSql) {
```

Comment:
I have a similar PR that attempts to purge more of the uses of raw `MakeReadWriteTransaction()` from the integration tests in favor of `RunTransaction()` (there are more in the samples that we can deal with later).  For example, from `SetUp()` and `MultipleInserts()`, in addition to `ExecuteSql()`.  [`TransactionRollback()` probably needs to deal with ABORTED by itself.]  Should we attempt to deal with those cases here?

Reply:
Fixed all the ones in `client_integration_test.cc` except for Rollback, I think we should deal with that and the examples in a future PR.